lib/delayed/backend/ironmq.rb

```diff
@@ -77,16 +77,18 @@ def save!
         end
 
         def destroy
-          if @msg
+          if @msg && @msg.reserved_count > @timeout_retries
+             @msg = ironmq.queue(queue_name).get_message(@msg.id)
```

Comment:
sorry, don't understand it clearly.
so, if condition is true, don't delete a message? what about next pick from the queue, that message is still there as far as i understand.